.travis.yml

```diff
@@ -0,0 +1,20 @@
+branches:
+  only:
+  - master
+  - hookshot-replacement
+  - hookshot-replacement-encrypted-key
+before_install:
+- openssl aes-256-cbc -K $encrypted_d5384b7469e5_key -iv $encrypted_d5384b7469e5_iv
+  -in $KEY_FILE.enc -out $KEY_FILE -d
```

Comment:
The name of this file isn't sensitive, so we don't gain much from maintaining it in an environment variable. (Looks like you might have come to a similar conclusion in https://github.com/bocoup/jqfundamentals.com/pull/15)